access-token-management/perf/Perf.DevServer/Perf.DevServer.csproj

```diff
@@ -1,19 +1,19 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <Sdk Name="Aspire.AppHost.Sdk" Version="9.0.0" />
+  <Sdk Name="Aspire.AppHost.Sdk" Version="9.3.1" />
```

Comment:
This should probably be 9.4.2 - latest version, matches the hosting package refs below...